src/main.cpp

```diff
@@ -1923,6 +1943,83 @@ bool CheckTxInputs(const CTransaction &tx, CValidationState &state, const CCoins
 }
 } // namespace Consensus
 
+bool CheckInputsAnalyzeTx(CTransaction &tx,
```

Comment:
maybe, but this code can just be deleted in the next version (once we are beyond the fork)